devices/common_patches/flexspi.yaml

```
@@ -0,0 +1,18 @@
+# Add SERCLKDIV to FLEXSPIx MCR0 registers 
+FLEXSPI*: 
+  MCR0: 
+    _add: 
+      SERCLKDIV: 
+        description: serial root clock divisor
+        bitOffset: 8
+        bitWidth: 10
```

The bitwidth is 3 bits. Confirmed in the 1060 and 1020 RMs.
```suggestion
        bitWidth: 3
```